core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -702,7 +703,11 @@ abstract class SnappyDDLParser(session: SparkSession)
           if (separatorIndex >= 0) {
             val key = rest.substring(0, separatorIndex).trim
             val value = rest.substring(separatorIndex + 1).trim
-            SetCommand(Some(key -> Option(value)))
+            if (key.startsWith("spark.sql.aqp.")) {
+              new SetCommand(Some(key -> Option(value))) with InvalidateCachedPlans
+            } else {
+              SetCommand(Some(key -> Option(value)))
+            }
```

Comment:
@ahshahid This is not the way we deal with actions on key set/unset. Code currently handles this in SnappyConf.keyUpdateActions so this should be moved there and the new InvalidateCachedPlans should be removed.